### DISCUSSION TITLE
微軟聊天機器人崇拜希特勒、色情、好戰

### DISCUSSION THREAD

**Discussion by 109302003** [1101-GS4719-C/Python-Community-GS4719-C#313] [General]
[https://www.ithome.com.tw/news/104851]() 

聊天機器人是透過機器學習，使用者與機器人講愈多話，它就會愈變愈聰明。在惡意網友的影響下，微軟的聊天機器人Tay發表了各種荒腔走板的言論，包括反女性主義者、種族仇恨、反猶太的言論，隨後只能下架此聊天機器人。

**Comment by 109802503gloria** [comment]
同學給的新聞連結url錯誤了喔，會導回這個討論串XD

**Reply by 109802503gloria** [reply]
其實我覺得教聊天機器人就跟小孩的教育是一樣的，在沒有分辨能力的情況下灌輸他什麼樣的內容會很大的影響他的價值觀。

**Comment by ChinChun0220** [comment]
這讓我想起國高中幼稚的line 聊天機器人，也可以透過keyin關鍵字並讓它學習回應，然後到後面就是一堆腥羶色的回應了…看到這個新聞完全不意外…

**Comment by mesakiyuta** [comment]
https://www.ettoday.net/dalemon/post/16069
應該是講同件事?

笑死，直接帶壞AI，他還嗆的很有力，換用其他社群軟體不知道學習成效會不會有差?

**Comment by cody910120** [comment]
就跟那個抽抽機機器人一樣好色喔(單論色情的部分

**Comment by JAIMEHUNG** [comment]
很像賴上面的卡米狗，但這個的語法用字更貼近人類

**Comment by yt7l0** [comment]
好好笑完全被教壞
不過他連打字的方式都很接近人類，學習力好強

**Comment by YuHengChen54** [comment]
這個機器人也太好笑，講話比我還嗆欸
